src/Tests/Integration/fixtures/banks/amex/02-pre-login/frame-3.html

```
@@ -5,7 +5,7 @@
   </head>
   <body style="background-color: transparent">
     <img
-      src="https://adservice.google.com/ddm/fls/z/src=16046083;type=rmrkt;cat=amexi0;rcb=16;ord=9857547701408;npa=0;auiddc=*;pscdl=noapi;frm=0;user_data_mode=a;_tu=KFA;gtm=45fe6631v9234171982z872195715za20gzb72195715zd72195715xea;gcd=13l3l3l3l1l1;dma=0;dc_fmt=2;tag_exp=0~115938466~115938468~118228214~119027222;epver=2;em=tv.*;dc_random=1780656156__VYDYyuTh1Ppvu0K4OYTXr2Z31gFH-8hCw;_dc_test=1;~oref=https%3A%2F%2Fhe.americanexpress.co.il%2Fpersonalarea%2Flogin%2F"
+      src="https://adservice.google.com/ddm/fls/z/src=16046083;type=rmrkt;cat=amexi0;rcb=16;ord=9857547701408;npa=0;auiddc=*;pscdl=noapi;frm=0;user_data_mode=a;_tu=KFA;gtm=45fe6631v9234171982z872195715za20gzb72195715zd72195715xea;gcd=13l3l3l3l1l1;dma=0;dc_fmt=2;tag_exp=0~[redacted-id]~[redacted-id]~[redacted-id]~[redacted-id];epver=2;em=tv.*;dc_random=1780656156__VYDYyuTh1Ppvu0K4OYTXr2Z31gFH-8hCw;_dc_test=1;~oref=https%3A%2F%2Fhe.americanexpress.co.il%2Fpersonalarea%2Flogin%2F"
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)